core.d.ts

```diff
@@ -24,35 +24,38 @@ export type FileTypeResult = {
 };
 
 /**
-Detect the file type of a `Uint8Array`, or `ArrayBuffer`.
+Detect the file type of `Uint8Array`, or `ArrayBuffer`.
```

Comment:
Don't do unrelated changes. `a` was correct.

```suggestion
Detect the file type of a `Uint8Array` or `ArrayBuffer`.
```